sklearn/ensemble/_hist_gradient_boosting/binning.py

```diff
@@ -40,7 +42,7 @@ def _find_binning_thresholds(data, max_bins, subsample, random_state):
     binning_thresholds: list of arrays
         For each feature, stores the increasing numeric values that can
         be used to separate the bins. Thus ``len(binning_thresholds) ==
-        n_features``.
+        n_features``. The first threshold (for missing values) is always NaN.
```

Comment:
Took me a while to understand this sentence, I was reading it in my brain as: "The first threshold for missing values is always NaN". Whereas I should have been reading it as "The first threshold corresponding to missing values is always NaN", I think.